ssl/s3_lib.c

```diff
@@ -834,6 +834,21 @@ static SSL_CIPHER ssl3_ciphers[] = {
      256,
      256,
      },
+    {
```

Comment:
and if we go down that route would we also have to put ifdef's around all the other ciphers for if you disabled everything *but* TLSv1.3? Doesn't sound nice :-(